### ISSUE TITLE
Add Multiplayer Feature

### ISSUE THREAD

**Issue by landry314** [lexica/lexica#217]
I'd love to see a multiplayer feature to play head to head with a friend on 2 different devices.

This could be implemented like the game in the play store called "just boggle" where it gives you a unique code for the current game you can share.

Or I imagine for games in the same room it could be done over bluetooth! This could allow for both competitive and real-time cooperative gameplay!

**Comment by pserwylo** [comment]
Thanks for the feedback. I'll mark this as a duplicate of #102, and we can continue discussing there.

**Comment by landry314** [comment]
Right, but how about a real time cooperative setup where you see the letters light up when your teammate gets a word and the points are shared?

**Comment by pserwylo** [comment]
Ah, makes sense. I'll open this again and update the title to reflect that it is for a real time network based multiplayer feature. Having said this, my first thoughts are that it would be quite a large feature to implement, so unless someone is able to contribute a significant amount of time it is unlikely to be implemented ahead of some of the other more pressing issues :)

**Comment by dirby7** [comment]
At the end of a multiplayer game, would love to see words we both found, words unique to each player, maybe in columns or color coded.

**Comment by dirby7** [comment]
It would be great to have either collaborative mode or competitive mode. In competitive mode, when both players get a word it is marked through so your score only includes words that only you got. 

Don't forget multiplayer can be more than 2 people.
